lib/widgets/content_list_view/song_tile.dart

```diff
@@ -256,16 +256,16 @@ class _SongTileState extends SelectableState<SelectionEntry<Song>, SongTile> wit
         title: title,
         subtitle: subtitle,
         leading: albumArt,
-        trailing: selectionRoute
-          ? Row(
-              mainAxisSize: MainAxisSize.min,
-              children: [
-                if (widget.trailing != null)
-                  widget.trailing!,
-                buildAddToSelection(),
-              ],
-            )
-          : widget.trailing,
+        trailing: Row(
```

Comment:
It feels like there is a unnecessarily large gap between the text and the heart indicator:
![Songs list with hearts](https://user-images.githubusercontent.com/6966049/169670331-7f9038b6-52f5-47e9-a0c8-e61685bf9a5d.png)

The gap comes from the padding of the `FavoriteIndicator`, but also the padding of the `ListTile`:
![Songs list with container markings](https://user-images.githubusercontent.com/6966049/169670332-97256c58-cd69-4b9e-ad39-5500a6a73a53.png)

We could reduce the `horizontalTitleGap` of the `ListTile`, but the better approach is probably to remove the padding from the `FavoriteIndicator`.

Reply:
Sorry, I'm not really sure what you are trying to tell me.

I also noticed that there is now more empty space to the right, even if the favorite indicator is not shown: (focus on the second and fourth list entry.)

| 1.0.8 | favorites |
|------|-----------|
| ![Song list - Before](https://user-images.githubusercontent.com/6966049/172019517-4ba442cd-6fa8-4db0-9da5-143e519ecff0.png) | ![Song list - After](https://user-images.githubusercontent.com/6966049/172019522-f85bd5b7-92cb-45f6-b613-d7f09b7d951f.png) |
 
Is this how you wanted it? In my opinion it would look better if there would be no empty space.